docs/products/cassandra/howto/disable-cross-cluster-replication.md

```diff
@@ -1,22 +1,19 @@
 ---
 title: Disable cross-cluster replication in Aiven for Apache Cassandra®
+pro: true
```

Comment:
Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise